docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -0,0 +1,636 @@
+            "Epoch 3/5\n",
+            "1852/1875 [============================>.] - ETA: 0s - loss: 0.4590 - accuracy: 0.8386\n",
+            "Epoch 00003: saving model to ./training\n",
+            "1875/1875 [==============================] - 4s 2ms/step - loss: 0.4590 - accuracy: 0.8385\n",
+            "Epoch 4/5\n",
+            "1874/1875 [============================>.] - ETA: 0s - loss: 0.4305 - accuracy: 0.8501\n",
+            "Epoch 00004: saving model to ./training\n",
+            "1875/1875 [==============================] - 4s 2ms/step - loss: 0.4306 - accuracy: 0.8500\n",
+            "Epoch 5/5\n",
+            "1866/1875 [============================>.] - ETA: 0s - loss: 0.4115 - accuracy: 0.8555\n",
+            "Epoch 00005: saving model to ./training\n",
+            "1875/1875 [==============================] - 3s 2ms/step - loss: 0.4114 - accuracy: 0.8556\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "<tensorflow.python.keras.callbacks.History at 0x7fab4c61d358>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 8
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uU2iQ6HAZ6-E",
+        "colab_type": "code",
+        "outputId": "07d0b8a1-666a-427b-ca54-54252061dc93",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 69
+        }
+      },
+      "source": [
+        "#Evalute results\n",
+        "model.load_weights(checkpoint_dir)\n",
+        "loss, accuracy = model.evaluate(test_images, test_labels, batch_size=32, verbose=2)\n",
+        "print(\"Loss :\", loss)\n",
+        "print(\"Accuracy :\", accuracy)"
+      ],
+      "execution_count": 9,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "10000/10000 - 0s - loss: 79.5507 - accuracy: 0.8011\n",
+            "Loss : 79.5506900449872\n",
+            "Accuracy : 0.8011\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lAvhD4unmc6W",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Moving Average SGD"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "--NIjBp-mhVb",
+        "colab_type": "code",
+        "outputId": "5db6d0e1-7baa-4406-ce20-a3f341e10e9f",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 471
+        }
+      },
+      "source": [
+        "#Build Model\n",
+        "model = create_model(moving_avg_sgd)\n",
+        "\n",
+        "#Train the network\n",
+        "model.fit(fmnist_train_ds, epochs=5, callbacks=[cp_callback])"
```

Comment:
```suggestion
        "model.fit(fmnist_train_ds, epochs=5, callbacks=[avg_callback])"
```
Is that intended that the normal callback is used for `moving_avg_sgd` or am I missing something?

Reply:
No I missed it.